### PR TITLE
applications: nrf_desktop: Assign desktop mice PID

### PR DESCRIPTION
**Pull request by MarekPieta**
Assigning new PIDs for desktop mice.

### CODE REVIEW

File: scripts/hid_configurator/configurator.py [hunk 1]
```diff
@@ -54,6 +49,14 @@
 DFU_SYNC_RETRIES = 3
 DFU_SYNC_INTERVAL = 1
 
+NORDIC_VID = 0x1915
+DEVICE_PID = {
+    'red_desktop_mouse'     : 0x52DA,
```
Comment (pdunaj) [inline scripts/hid_configurator/configurator.py]: desktop_mouse_nrf52832

File: scripts/hid_configurator/configurator.py [hunk 2]
```diff
@@ -54,6 +49,14 @@
...
+    'black_desktop_mouse'   : 0x52DB,
```
Comment (pdunaj) [inline scripts/hid_configurator/configurator.py]: desktop_mouse_nrf52810

Comment (MarekPieta) [reply scripts/hid_configurator/configurator.py]: done

File: scripts/hid_configurator/configurator.py (hunk 1 above)
Comment (MarekPieta) [reply scripts/hid_configurator/configurator.py]: done

**Pull request merged**